Dockerfiles/local-gadget-tests.Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.18 AS builder
+
+RUN \
+	dpkg --add-architecture arm64 && \
+	apt-get update && \
+	apt-get install -y gcc-aarch64-linux-gnu build-essential crossbuild-essential-arm64 && \
+	apt-get install -y libseccomp2       libseccomp-dev && \
+	apt-get install -y libseccomp2:arm64 libseccomp-dev:arm64
+
+COPY go.mod go.sum /cache/
+RUN cd /cache && go mod download
+
+ADD . /go/src/github.com/inspektor-gadget/inspektor-gadget
+WORKDIR /go/src/github.com/inspektor-gadget/inspektor-gadget/integration/local-gadget
```

Comment:
How about moving them to a common builder image (say `lg-builder`)? both the images can still use `lg-builder` and have independent `dockerignorefiles`. (maybe [`ONBUILD`](https://docs.docker.com/engine/reference/builder/#onbuild) in useful for `COPY/RUN/ADD`). Do you think it will be useful?

Reply:
Did you mean it in this way?

Reply:
Yep. This is looking better to me. These lines can be removed from here I reckon?